AnkiHubSoftware/ankihub_addon
# [BUILD-920] feat: Adjust the behavior of the sidebar buttons according to the user settings

## Related issues
- [BUILD-920](https://ankihub.atlassian.net/browse/BUILD-920)


## Proposed changes
This PR ensures that each sidebar button will appear only if their option on the configuration menu is selected.


## How to reproduce
1. Go to the ankihub configuration menu, go to the sidebar section, and check the options of the buttons that you want to show in the sidebar of the reviewer
2. Go to the card reviewer
3. Make sure that the buttons are being shown according to the selected options (not considering other rules like the feature flags)

[BUILD-920]: https://ankihub.atlassian.net/browse/BUILD-920?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
